packages/liveblocks-react-comments/src/styles/index.css

```diff
@@ -639,6 +639,12 @@
   }
 }
 
+/* 0,0,0 specificity to inherit any styles applied to `code` elements */
```

Comment:
Here's before and after on Windows

![image](https://github.com/liveblocks/liveblocks/assets/33033422/1c8d5125-2ee8-43c1-ad50-f757b077e2eb)